addons/mail/static/src/components/attachment_viewer/attachment_viewer.scss

```diff
@@ -3,134 +3,37 @@
 // ------------------------------------------------------------------
 
 .o_AttachmentViewer {
-    display: flex;
-    width: map-get($sizes, 100);
-    height: map-get($sizes, 100);
-    flex-flow: column;
-    align-items: center;
     z-index: -1;
 }
 
 .o_AttachmentViewer_buttonNavigation {
-    position: absolute;
-    display: flex;
-    align-items: center;
-    justify-content: center;
     width: 40px;
     height: 40px;
-    top: 50%;
-    transform: translateY(-50%);
 }
 
-.o_AttachmentViewer_buttonNavigationNext {
-    right: 15px;
-
-    > .fa {
-        margin: 1px 0 0 1px; // not correctly centered for some reasons
-    }
+.o_AttachmentViewer_buttonNavigationNext > .fa {
```

Comment:
Don't deal with CSS specificity. Introduce classname `.o_AttachmentViewer_buttonNavigationNextIcon` on this element, and just target with this classname.